cmd/notifier/main.go

```diff
@@ -148,6 +148,10 @@ func stopNotificationsFetcher(worker *notifications.FetchNotificationsWorker) {
 
 func stopSelfStateChecker(checker *selfstate.SelfCheckWorker) {
 	if err := checker.Stop(); err != nil {
-		logger.Errorf("Failed to stop self check worker: %v", err)
+		if errors.Is(err, selfstate.ErrDisabled) {
+			logger.Warning(err.Error())
```

Comment:
oops :) 

lets make logger.Warning(err)

Reply:
There are `logger.Warning` and `Errorf` inside if statement. I think it is enough

Reply:
no, in first you use err.Error() (however logger.Warning use interface)

and second use "err"

lets make it the same type